### ISSUE TITLE
tfswitch --help or --version can not run unless you have write access to /usr/local/bin/terraform

### ISSUE THREAD

**Issue by rblumen-desk** [warrensbox/terraform-switcher#55]
If /usr/local/bin/terraform exists (does not have to be in your PATH) and you do not have write access to the directory or file, then you can not run even commands that arguably should be non-destructive - e.g. tfswitch --help. 

This is due to the lib.init() method which tries to unlink that path and runs before the command arguments are processed.   I think that there should be a way to override or disable this behavior if you don't want it to run. 

A work around is to create an executable file named terraform, put it in a different directory that you have write access to, and then add that directory to your PATH.

**Comment by evansj** [comment]
I think this bug also hits you if you use a custom bin path.

```bash
➜  ~ tfswitch -b ~/bin/terraform 0.12.13
Switched terraform to version "0.12.13"
➜  ~ ls -l ~/bin/terraform
lrwxr-xr-x  1 jevans  staff  51  7 Nov 11:58 /Users/jevans/bin/terraform@ -> /Users/jevans/.terraform.versions/terraform_0.12.13
➜  ~ tfswitch -v

Version: 0.7.737
➜  ~ ls -l ~/bin/terraform
ls: /Users/jevans/bin/terraform: No such file or directory
```

I'm not even sure how that final invocation of `tfswitch` even knows about the custom bin path because I don't have a `.tfswitchrc` or `.tfswitch.toml`.

**Comment by warrensbox** [comment]
@evansj you need to create the `~/bin/terraform` prior to setting that path as your bin directory. I have included an error message to help users int the future. Thanks for bringing this up!

**Comment by warrensbox** [comment]
@rblumen-desk 
Fixed with #68
`brew upgrade warrensbox/tap/tfswitch` to get the latest version
